src/plugins/ucx/ucx_backend.cpp

```
@@ -51,38 +52,33 @@ class nixlUcxBackendH : public nixlBackendReqH {
-public:
-    nixlUcxBackendH(nixlUcxWorker *worker, size_t worker_id)
-        : worker(worker),
-          worker_id(worker_id) {}
+    std::optional<Notif> notif;
 
-    auto &
-    notification() {
-        return notif;
-    }
+    nixlUcxBackendReqH(nixlUcxWorker *worker, size_t worker_id)
+        : worker_(worker),
```

Cppcheck (2.20.0)

[style] The function 'getMem' is never used.

(unusedFunction)